tests/plugins/clipboard/manual/pasteimage.md

```diff
@@ -1,10 +1,16 @@
 @bender-ui: collapsed
-@bender-tags: 4.6.2, bug, clipboard
+@bender-tags: 4.6.2, 4.17.0, bug, clipboard, 3757
 @bender-ckeditor-plugins: wysiwygarea, toolbar, undo, basicstyles, image, clipboard, sourcearea
 
 ## Scenario:
 
- * Paste image from your clipboard into the editor.
```

Comment:
> I didn't add new unit tests as it seems that we are covered enough in this matter.

Well it is more related to https://github.com/ckeditor/ckeditor4/pull/4619, and maybe it is intended, but: 
1. right-click at any internet graphic and use the "copy image" option (name probably depends on browser)
2. try to "Paste image from your clipboard" image is pasted as URL, not base64 :thinking: 

I find that there is an additional type in types when `copy/paste` like that and it is "text/html" but also "Files" is present there. Is it some edge case or I miss some knowledge?

Reply:
That's actually an interesting question :thinking: I don't really know, what is an intended behaviour. But I suspect if the clipboard contains the copied images as HTML `img` element, there is no need to convert it into Base64.